trunk/SUAVE/Methods/Aerodynamics/Common/Fidelity_Zero/Lift/extract_wing_VD.py

```diff
@@ -0,0 +1,67 @@
+## @ingroup Methods-Aerodynamics-Common-Fidelity_Zero-Lift
+# extract_wing_collocation_points.py
+#
+# Created:   Feb 2022, R. Erhard
+# Modified: 
+
+# ----------------------------------------------------------------------
+#  Imports
+# ----------------------------------------------------------------------
+
+# package imports
+import numpy as np
+from SUAVE.Core import Data
+
+## @ingroup Methods-Aerodynamics-Common-Fidelity_Zero-Lift
+def extract_wing_collocation_points(geometry, wing_instance_idx):
+
+    """ This extracts the collocation points of the vehicle vortex distribution
+    belonging to the specified wing instance index. This is used for 
+
+    Source:
+    None
+
+    Inputs:
+    geometry      -    SUAVE vehicle 
+    wing_instance -    wing instance to extract VD for
+
+    Outputs:
+    None
+
+    Properties Used:
+    N/A
+    """
+    # unpack vortex distribution properties
+    VD   = geometry.vortex_distribution
+    n_sw = VD.n_sw 
+    n_cw = VD.n_cw  
+
+    VD_wing = Data()    
+    vd_i    = 0        # count of current VD wing elements
+    j       = 0        # count of current VD wing index
+    size = n_cw * n_sw
+    
+    for idx,wing in enumerate(geometry.wings):
```

Comment:
I'm confused why you even need this whole function? Maybe we can talk about indexing through the VLM. Aaron and I have a nice way of splitting out wings if that's what this is trying to do.

Reply:
The VLM includes the indices of where a wing stops and the next begins. [You can see an example here where it splits](https://github.com/suavecode/SUAVE/blob/develop/trunk/SUAVE/Methods/Aerodynamics/Common/Fidelity_Zero/Lift/VLM.py#L480). Just remember the VLM works on semispan wings.